### DISCUSSION TITLE
RTM with field data is pretty noisy

### DISCUSSION THREAD

**Discussion by kerim371** [slimgroup/JUDI.jl#160] [Q&A]
Hi,

Can't undrstand why am I getting so noisy RTM image.
Wavefield segy file with muted diving waves.
Velocity model spacing is 6.25 m and it was computed using FWI with 14 Hz at maximum.
Source Ricker wavelet at 25 Hz.
Scalar propagator (velocity only).
```julia
# JUDI options
jopt = JUDI.Options(
    limit_m = true,
    buffer_size = 1000f0,
    optimal_checkpointing=false,
    IC = "isic")

# Setup operators
Pr = judiProjection(d_obs.geometry)
F = judiModeling(model0; options=jopt)
Ps = judiProjection(q.geometry)
J = judiJacobian(Pr*F*adjoint(Ps), q)

# RTM
rtm = adjoint(J)*d_obs
```

JUDI v3.1.13

![image](https://user-images.githubusercontent.com/43808863/203055910-a2f8a74c-ee7a-4575-a9ba-7471cc1484f2.png)
![image](https://user-images.githubusercontent.com/43808863/203056330-208ba253-8652-4297-8804-1666ac196ac6.png)
![image](https://user-images.githubusercontent.com/43808863/203055941-af24b671-b1c9-4a2f-8aa7-369595edf665.png)

**Comment by mloubout** [comment]
A lot of details can lead to noise/artifacts such as:
- Incorrect velocity (i.e background isn't estimated well enough(
- Incorrect wavelet
- Numerical dispersion
- Muting. Your data looks like it contains a hard mute (cuts directly to zero) which is not good for propagation, some type of cosine taper is usually needed
- Source/rec imprint in the shallow area. It looks like you have coarse source/rec and therefore see the acquisition imprint.

So I would check all these details as field-data imaging is very sensitive to any small issue. The FWI velocity also looks like it contains artifacts so I would try to invert for a better velocity as well.

**Comment by kerim371** [comment]
@mloubout before that I used to calculate FWI based on data after multiples removal. But now I got better results calculated on raw data (only gain was added). FWI calculated on frequencies: 3, 5, 8, 14 Hz:
![image](https://user-images.githubusercontent.com/43808863/204543860-c9c3ad0b-0a3b-459f-8178-5a293169f2f7.png)

Then I use this model to calculate RTM image and I still get bad result (somthing similar to the one mentioned above).

The main questions I can't answer:
1) is it fine for JUDI to calculate FWI on data with multiples?
2) for FWI the model's grid size is equal to CMP spacing: 12.5x12.5 meters. It is sufficient to steadily calculate wavefields with signals up to 14 Hz with the given model. Increasing the frequency leads to artifacts in forward modeling. Is it a good practice to interpolate the model (make spacings 6.25x6.25 m) to calculate FWI up to 25 Hz for example? Or what is the maximal frequency for calculating FWI usually?
3) before calculating RTM should we remove multiples, direct and refracted/diving waves or should we leave them present in the data?

**Reply by mloubout** [reply]
1. Yes you can run FWI with multiples with JUDI, just make sure that you run it with `Options(free_surface=true)` so that you model the multiples as well
2. You need a grid that avoids dispersion. For the lower frequencies 12.5m is definitely enough, at 14Hz it may start to get dispersive so you'll need to either resample on a finer grid or use higher order (Options(space_order=16) for example) to reduce the dispersion (or a combination of both)
3. RTM is usually looking much better if you remove the multiples and mute the diving waves and only image the reflections.

**Reply by mloubout** [reply]
If you add gain to your data you need to add the same gain to the JUDI modelled data as well. Also remember that this is field, therefore 3D data, so you may need to do some extra data matching step to correct for the different geometrical spreading in 2D

**Comment by kerim371** [comment]
@mloubout right after I've set `free_surface=true` the FWI quality increased (5Hz->8Hz->14Hz on the pic):
![image](https://user-images.githubusercontent.com/43808863/204828300-5dd3dc8f-58c4-4e79-b6ff-e1a4e37fcfc9.png)

And RTM was also improved:
![image](https://user-images.githubusercontent.com/43808863/204829173-abdd9865-c06c-4dbc-9673-28a34e0e8bd0.png)

Thank you very much for your help!

P.S. low frequencies at the top (0.5-0.7 km) of RTM most likely caused by my decision not to mute first breaks (in learning purpose).

**Reply by mloubout** [reply]
Great, RTM looks much nicer as well.  The free surface incurs a phase shift (vertical derivative) so if you don't model it, it will definitely lead to much worse results.
